src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturemedia.controllers;
+
+import java.util.*;
+
+import culturemedia.exception.VideoNotFoundException;
+import culturemedia.model.Video;
+import culturemedia.service.impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+	private final CultureMediaServiceImpl cultureMediaService;
+
+
+	public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+		this.cultureMediaService = cultureMediaService;
+	}
+
+
+	public List<Video> find_allVideos() throws VideoNotFoundException {
+		List<Video> videos = null;
+		videos = cultureMediaService.findAll();
```

Comment:
It is not necessary to initialize **videos** as **null** if you are then going to override it with the result of `cultureMediaService.findAll()`. You can initialize it directly with the result of the method: `List<Video> videos = cultureMediaService.findAll()`